eslint.config.js

```diff
@@ -271,7 +279,7 @@ module.exports = [
 			"test/node_smoke_tests/**",
 			"test/bundler_smoke_tests/**/*",
 			"test/promises_aplus_adapters/**",
-			"test/middleware-mockserver.js"
+			"test/middleware-mockserver.cjs"
```

Comment:
I think we had said it'd be nice if this filename matched the main branch, so I've made that change in this PR.